docs/static/releasenotes.asciidoc

```diff
@@ -3,6 +3,8 @@
 
 This section summarizes the changes in the following releases:
 
+* <<logstash-7-3-0,Logstash 7.3.0>>
+* <<logstash-7-2-0,Logstash 7.2.0>>
```

Comment:
Do we need 7.2.1?

Reply:
:thinking: I honestly wouldn't expect any of the patch-level releases to be included in a changelog for a minor-level release, and am a bit surprised to see them listed here. The changelog entry below for 7.3.0 includes fixes that _also_ target a patch-level release on 7.2 (which hasn't been released), so I don't think there is risk in omitting it.